patches/009-add-sxtsq.patch

```diff
@@ -15,7 +15,8 @@ Index: openwrt/target/linux/ar71xx/base-files/lib/ar71xx.sh
 +	*"RBSXTsq2nD")
 +		name="rb-lhg-5nd"
```

Comment:
should this be:  name="rb-lhg-2nd" ??

Reply:
This section is untouched before/after in this commit.    All of the SXT 2 and 5 devices function with the same hardware specification, so when they were first added, I did not create new board ids and reused the lhg-5nd.  The settings are otherwise detected of the model string.  At some point,  probably going into ath79 target, we should be able to inherit all of the definitions from upstream openwrt, and this will go away.  We had added these models before openwrt.

Reply:
note the pre-existing sxt board ids were earlier hardware revisions that are not currently supported (the octagon form factor instead of the newer square form factor devices).